app/models/chargeback/consumption_with_rollups.rb

```diff
@@ -10,7 +10,7 @@ def initialize(metric_rollup_records, start_time, end_time)
     end
 
     def tag_names
-      first_metric_rollup_record.tag_names.split('|')
+      @tag_names ||= @rollups.map { |m| m.tag_names.split('|') }.flatten.uniq
```

Comment:
I think that some `tag_names`  on the rollup can be nil, so probably we need something like:
```ruby
@rollups.where.not(:tag_names => nil ).map { |m| m.tag_names.split('|') }.flatten.uniq
```